pkg/plugin/v3/scaffolds/webhook.go

```diff
@@ -86,5 +85,15 @@ You need to implement the conversion.Hub and conversion.Convertible interfaces f
 		return err
 	}
 
+	// TODO: Add test suite for conversion webhook after #1664 has been merged & conversion tests supported in envtest.
```

Comment:
Why is this a TODO? Aren't you just doing this?

Reply:
This todo is because of #1664. 
Just to make clear that it should be updated after that as well.